### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Pravin-Kumar-21**
I have edited code line no  182 and 183 in which the variable was not initialised with the data types .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ i32 main {
 #include <stdio.h>
 #include <stdlib.h>
 int main(int argc, char *argv[]) {
-   global_argc = argc;
+  int global_argc = argc;
```
Comment (jbampton) [inline README.md:159]:
```suggestion
   int global_argc = argc;
```

File: README.md [hunk 2]
```diff
@@ -179,8 +179,8 @@ main {
 #include <stdlib.h>
 #include <string.h>
 int main(int argc, char *argv[]) {
-   global_argc = argc;
-   global_argv = argv;
+   int global_argc = argc;
+  char * global_argv = *argv;
```
Comment (jbampton) [inline README.md:183]:
```suggestion
   char * global_argv = *argv;
```

**Comment by BaseMax** [comment]
@jbampton please review it.

**Review by jbampton — COMMENTED** [review]
You added `int` in some places but not all.

Also `global_argv = argv;`  was only changed in one place.

I am not a C expert but seems inconsistent

**Pull request closed without merging**